vendor/github.com/ethereum/go-ethereum/core/headerchain.go

```diff
@@ -40,6 +42,21 @@ const (
 	numberCacheLimit = 2048
 )
 
+var writeDelay time.Duration = 0
+
+func init() {
+	str, ok := os.LookupEnv("FEATURE_SYNC_DELAY")
+	if !ok {
+		return
+	}
+	delay, err := strconv.ParseInt(str, 10, 0)
+	if err != nil {
+		panic(fmt.Errorf("FEATURE_SYNC_DELAY value is invalid:", err))
```

Comment:
`panic()` does not take a string as its argument?

Reply:
@adambabik hence `fmt.Errorf()` call :)

Reply:
@adambabik hence `fmt.Errorf()` call :)